File: eTeatar/eTeatar.Model/Vijest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eTeatar.Model
{
    public class Vijest
    {
        public int VijestId { get; set; }

        public string Naziv { get; set; }

        public string Sadrzaj { get; set; } 

        public DateTime Datum { get; set; }

        public int KorisnikId { get; set; }
    }
}

File: eTeatar/UI/eteatar_desktop/lib/models/vijest.dart
import 'package:json_annotation/json_annotation.dart';

part 'vijest.g.dart';

@JsonSerializable()
class Vijest {
  int? vijestId;
  String? naziv;
  String? sadrzaj;
  DateTime? datum;
  int? korisnikId;


  Vijest(
    this.vijestId,
    this.naziv
  );

  factory Vijest.fromJson(Map<String, dynamic> json) => _$VijestFromJson(json);

  /// Connect the generated [_$GlumacToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$VijestToJson(this);
}